addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -50,14 +49,17 @@ type MirrorPeerReconciler struct {
 }
 
 const (
-	RookCSIEnableKey              = "CSI_ENABLE_OMAP_GENERATOR"
-	RookVolumeRepKey              = "CSI_ENABLE_VOLUME_REPLICATION"
-	MirroringModeKey              = "mirroringMode"
-	SchedulingIntervalKey         = "schedulingInterval"
-	ReplicationSecretNameKey      = "replication.storage.openshift.io/replication-secret-name"
-	ReplicationSecretNamespaceKey = "replication.storage.openshift.io/replication-secret-namespace"
-	ProvisionerTemplate           = "%s.rbd.csi.ceph.com"
-	RookConfigMapName             = "rook-ceph-operator-config"
+	RookCSIEnableKey                   = "CSI_ENABLE_OMAP_GENERATOR"
+	RookVolumeRepKey                   = "CSI_ENABLE_VOLUME_REPLICATION"
+	MirroringModeKey                   = "mirroringMode"
+	SchedulingIntervalKey              = "schedulingInterval"
+	ReplicationSecretNameKey           = "replication.storage.openshift.io/replication-secret-name"
+	ReplicationSecretNamespaceKey      = "replication.storage.openshift.io/replication-secret-namespace"
+	ProvisionerTemplate                = "%s.rbd.csi.ceph.com"
+	RookConfigMapName                  = "rook-ceph-operator-config"
+	VolumeReplicationClassNameTemplate = "odf-rbd-volumereplicationclass-%v"
+	ReplicationSecretName              = "rook-csi-rbd-provisioner"
```

Comment:
RBDReplicationSecretName.

Reply:
will this name remain the same for external mode also?

Reply:
Yes currently we are keeping it the same. We are not supporting external mode for now.